edgekv/readme.md

```diff
@@ -0,0 +1,19 @@
+# Akamai EdgeKV Examples
+Akamai Edge Key-Value (EdgeKV) is a low-latency, distributed key-value store built to compliment EdgeWorkers. EdgeKV enables you to build data-driven JavaScript applications that require fast, frequent reads and infrequent writes. The code snippets in this repository represent a jumping-off point to help you learn about EdgeKV, and how it integrates with EdgeWorkers.

```

Comment:
Can you take a look at the formatting we use for READMEs in other EW examples and ensure the same sections are available here?

Reply:
Agreed to update in subsequent PR.